dailysheet.php

```diff
@@ -956,6 +956,18 @@ function fieldchange(what, row = null) {
 
   var charges = document.getElementById("k" + iRow).value;
   var comments = document.getElementById("l" + iRow).value;
+  
+  //HACK:GWR - BEGIN
+  if (comments.toUpperCase().includes("GWR")){
+      var ch = document.getElementById("k" + iRow).childNodes;
+      for (mm = 0; mm < ch.length; mm++) {
+        ch[mm].selected = false;
+        if (ch[mm].value == "c13")
+          ch[mm].selected = true;
+      }	
+  }
+  //HACK:GWR - END
```

Comment:
This is to catch when users not aware of this new functionality will keep adding the "GWR" comment: the system will set the type to "Charge GWR" automatically. 

The hope is to get rid of it at some point, when all the users will be trained.